src/main/webapp/WEB-INF/web.xml

```diff
@@ -20,28 +20,25 @@
 
 -->
 <web-app id="jasig-utility-portlets" version="2.4" xmlns="http://java.sun.com/xml/ns/j2ee"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd">
-	<display-name>Jasig Utility Portlets</display-name>
-
-	<!--
-		| Key of the system property that should specify the root directory of this
-		| web app. Applied by WebAppRootListener or Log4jConfigListener.
-		+-->
-	<context-param>
-		<param-name>log4jConfigLocation</param-name>
-		<param-value>/WEB-INF/log4j.properties</param-value>
-	</context-param>
```

Comment:
Does SLF4J also need to be pointed towards a config file?

Reply:
As far as I am aware of, SLF4J is an abstraction layer. Logback is the library that does logging and a [logback,xml](https://github.com/uPortal-Project/JasigWidgetPortlets/blob/master/src/main/resources/logback.xml) config file does exist for it.